### PR TITLE
Update project-resource-quotas.md

### PR DESCRIPTION
**Pull request by mitovskaol**
Updated to reflect the new quota sizes

### CODE REVIEW

File: docs/resourceTuning/project-resource-quotas.md [hunk 1]
```diff
@@ -14,24 +14,60 @@ tags:
...
+storage-2 Overall Storage: 2 GiB, Backup Storage: 1 GiB
+storage-4 Overall Storage: 4 GiB, Backup Storage: 2 GiB
+storage-16 Overall Storage: 16 GiB, Backup Storage: 8 GiB
+storage-32 Overall Storage: 32 GiB, Backup Storage: 16 GiB
+storage-64 Overall Storage: 64 GiB, Backup Storage: 32 GiB
+storage-128 Overall Storage: 128 GiB, Backup Storage: 64 GiB
+storage-256 Overall Storage: 256 GiB, Backup Storage: 128 GiB
+storage-512 Overall Storage: 512 GiB, Backup Storage: 256 GiB
+```
+**All Storage Quotas include 60 PVCs**.
```
Comment (StevenBarre) [inline docs/resourceTuning/project-resource-quotas.md:73]:
```suggestion

**All Storage Quotas include 60 PVCs and 5 VolumeSnapshots**.
```

**Review by StevenBarre — APPROVED** [review]

**Review by j-pye — APPROVED** [review]
I also like Steven's suggestion. Looks good otherwise though. 👍🏻

File: docs/resourceTuning/project-resource-quotas.md [hunk 2]
```diff
@@ -14,24 +14,60 @@ tags:
...
 ---
 
-All project sets in the new OpenShift Container Platform (OCP) 4 Platform are provisioned by default with a "small" resource quota size that includes a certain amout of CPU, RAM and Storage.  The amount of resources within each quota size - small, medium and large are as follows:
+All project sets in the new OpenShift Container Platform (OCP) 4 Platform are provisioned by default with a set of resource quotas that include a certain amout of CPU, RAM and Storage:
 
 **CPU/RAM/Storage resource quotas (per namespace)**:
-* **Small (Default)**:<br/> 
   CPU: 4 cores as request, 8 cores as limit <br/>
   RAM: 16GBs as request, 32GBs as limit<br/>
   20 PVC count , 50Gbs overall storage with 25 GBs for backup storage<br/>
```
Comment (w8896699) [inline docs/resourceTuning/project-resource-quotas.md:22]: Our default size will have only 0.5-1.5 CPU, 2-4GiB Memory, and 1Gi Storage, 521m backup, 60 PVC and 5 snapshot

File: docs/resourceTuning/project-resource-quotas.md [hunk 3]
```diff
@@ -14,24 +14,60 @@ tags:
...
-CPU: 8 cores as request, 16 cores as limit<br/>
-RAM: 32GBs as request, 64GBs as limit<br/>
-Medium: 40 PVC count , 100Gbs overall storage with 50 GBs for backup storage<br/>
 
-* **Large: (needs to be requested and justified):**
-Long-running workload quotas:<br/>
-CPU: 16 cores as request, 32 cores as limit<br/>
-RAM: 64GBs as request, 128GBs as limit<br/>
-Large: 60 PVC count , 200Gbs overall storage with 100 GBs for backup storage 
+**If your app is fount not using all resources within the current quota, you will be approached by the Platform Services Team to discuss the downgrade to a smaller quota size.**
```
Comment (mitovskaol) [inline docs/resourceTuning/project-resource-quotas.md:25]:
```suggestion
**If your app is found not using all resources within the current quota, you will be approached by the Platform Services Team to discuss the downgrade to a smaller quota size.**
```

**Pull request merged**